### PR TITLE
Updated the links

### PR DESCRIPTION
**Pull request by Sayanta66**
Updated a link in the doc

### CODE REVIEW

**Comment by harshilsharma63** [comment]
I'm not sure which is the correct link for this one. @chenilim @jespino @michelengelen can you verify this?

**Comment by michelengelen** [comment]
@harshilsharma63 I am sorry, but I cannot help you with that as well.

**Comment by harshilsharma63** [comment]
> @harshilsharma63 I am sorry, but I cannot help you with that as well.

My bad, I intended to tag Miguel :P

File: website/site/content/download/mattermost/_index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The Focalboard plugin requires websocket traffic to be passed by the proxy. Upda
 
 ### With NGINX
 
-After following the standard [Mattermost install steps](https://docs.mattermost.com/install/installing-ubuntu-1804-LTS.html#configuring-nginx-as-a-proxy-for-mattermost-server), edit `/etc/nginx/sites-available/mattermost` and add this section to it:
+After following the standard [Mattermost install steps](https://docs.mattermost.com/configure/configuring-apache2.html#configuring-apache2-as-a-proxy-for-mattermost-server-unofficial), edit `/etc/nginx/sites-available/mattermost` and add this section to it:
```
Comment (jespino) [inline website/site/content/download/mattermost/_index.md:24]: Why this link changes to point to the apache2 configuration?

**Review by jespino — CHANGES_REQUESTED** [review]
The new link provided points to the wrong configuration page (we are talking about nginx and the new url points to the unofficial doc for configuring apache).

File: website/site/content/download/mattermost/_index.md (hunk 1 above)
Comment (Sayanta66) [reply website/site/content/download/mattermost/_index.md:24]: okay okay there has been a mishap....don't have to add this change

**Comment by Sayanta66** [comment]
I am closing this PR, because there have been a mishap in the link because I put a wrong link

**Pull request closed without merging**